### PR TITLE
PICNIC-864 - Decrypt/Verify output shows senders full name

### PR DESCRIPTION
**Pull request by thebearjew**
1. Adds a helper function to `stories/storybook.shared` for updated the store value in `Sb.MockStore` provider. This is really useful when working with components using hooks (`useSelector`). It allows for changing the store that components pull from when rendered in the Storybook context.
2. Changed the design of SignedSender
3. Renamed `outputSender` → `outputSenderUsername`
4. Added `outputSenderFullname`

image

cc @keybase/react-hackers @keybase/picnicsquad

### CODE REVIEW

File: shared/crypto/output/index.tsx [hunk 1]
```diff
@@ -42,44 +42,67 @@ export const SignedSender = (props: SignedSenderProps) => {
...
 
   return (
     <Kb.Box2 direction="horizontal" fullWidth={true} alignItems="center" style={styles.signedContainer}>
-      <Kb.Box2 direction="horizontal" gap="xtiny" alignItems="center" style={styles.signedSender}>
-        {signed && signedBy
-          ? [
-              <Kb.Avatar key="avatar" size={16} username={signedBy.stringValue()} />,
-              <Kb.Text key="signedBy" type="BodySmall">
-                Signed by {canSelfSign ? ' you, ' : ''}
+      {signed && signedByUsername ? (
```
Comment (buoyad) [inline shared/crypto/output/index.tsx]: This jsx is hard to follow, I wonder if it could be factored into separate components some. No need to change now.

Comment (thebearjew) [reply shared/crypto/output/index.tsx]: Yeah you're right. I'll clean it up and then merge. thanks!
